CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ## [Unreleased]
 ### Added
+- Enhanced Generator Code to generate Endpoints using OpenSearch API Specifications ([#194](https://github.com/opensearch-project/opensearch-php/pull/194))
```

Comment:
Nit: "Enhanced Generator Code " is kinda redundant, simplify "Generate endpoints from OpenSearch OpenAPI spec" or something like that.